backend/app/api/endpoints/recommendation.py

```diff
@@ -33,6 +35,24 @@ async def generate_recommendation(
         )
 
 
+@router.post("/additional_recommendations")
```

Comment:
Now the route will be `/recommendation/additional_recommendation` which is not very good design.
Maybe we can change the path to `recommendation/prompt`

And the function name can be `generate_prompt_recommendation`